src/build/GhosttyResources.zig

```diff
@@ -224,16 +224,76 @@ pub fn init(b: *std.Build, cfg: *const Config) !GhosttyResources {
         // https://developer.gnome.org/documentation/guidelines/maintainer/integrating.html
 
         // Desktop file so that we have an icon and other metadata
-        try steps.append(&b.addInstallFile(
-            b.path("dist/linux/app.desktop"),
-            "share/applications/com.mitchellh.ghostty.desktop",
-        ).step);
+        try steps.append(
+            formatService(
+                b,
+                cfg,
+                b.path("dist/linux/app.desktop"),
+                b.fmt(
+                    "share/applications/com.mitchellh.ghostty{s}.desktop",
+                    .{
+                        switch (cfg.optimize) {
+                            .Debug, .ReleaseSafe => "-debug",
+                            .ReleaseFast, .ReleaseSmall => "",
+                        },
+                    },
+                ),
+            ),
+        );
+        // Service for DBus activation.
+        try steps.append(
+            formatService(
+                b,
+                cfg,
+                b.path("dist/linux/dbus.service"),
+                b.fmt(
+                    "share/dbus-1/services/com.mitchellh.ghostty{s}.service",
+                    .{
+                        switch (cfg.optimize) {
+                            .Debug, .ReleaseSafe => "-debug",
+                            .ReleaseFast, .ReleaseSmall => "",
+                        },
+                    },
+                ),
+            ),
+        );
+        // systemd user service
+        if (!cfg.flatpak)
+            try steps.append(
+                formatService(
+                    b,
+                    cfg,
+                    b.path("dist/linux/systemd.service"),
+                    b.fmt(
+                        "{s}/systemd/user/com.mitchellh.ghostty{s}.service",
+                        .{
+                            if (cfg.system_package) "lib" else "share",
+                            switch (cfg.optimize) {
+                                .Debug, .ReleaseSafe => "-debug",
```

Comment:
Is there any precedent for the `-debug`? Should we follow Flatpak here and use `.Debug`?

Reply:
Yes, it's based on the DBus name Ghostty uses when compiled in debug mode so we don't have much of a choice.

Reply:
We could also just patch the used app-id, it's coded in only one place. But I have no issues with `-debug` suffix.

If we decide to patch that, I'd recommend `.Devel` or something similar, because `.Debug` is used for debug information package generated by flatpak.